### DISCUSSION TITLE
LibreOffice

### DISCUSSION THREAD

**Discussion by JuliaAgostino** [mayrinck/FOSSonMicrosoftStore#5] [Requests] [answered]
LibreOffice website: https://www.libreoffice.org/

LibreOffice privacy policy: https://www.libreoffice.org/about-us/privacy/

Logo: https://upload.wikimedia.org/wikipedia/commons/c/c0/LibreOffice_logo.svg

Description: "LibreOffice is a free and open-source office productivity software suite, a project of The Document Foundation (TDF). It was forked in 2010 from OpenOffice.org, which was an open-sourced version of the earlier StarOffice. The LibreOffice suite consists of programs for word processing, creating and editing of spreadsheets, slideshows, diagrams and drawings, working with databases, and composing mathematical formulae. It is available in 115 languages.

As its native file format to save documents for all of its applications, LibreOffice uses the Open Document Format for Office Applications (ODF), or OpenDocument, an international standard developed jointly by the International Organization for Standardization (ISO) and the International Electrotechnical Commission (IEC). LibreOffice also supports the file formats of most other major office suites, including Microsoft Office, through a variety of import and export filters.

LibreOffice is available for a variety of computing platforms, including Microsoft Windows, macOS, Linux, Android, iOS, and Chromebook. It is also available as an online office suite called LibreOffice Online, which includes the applications Writer, Calc and Impress. LibreOffice is the default office suite of most popular Linux distributions. It is the most actively developed free and open-source office suite, with approximately 50 times the development activity of Apache OpenOffice, the other major descendant of OpenOffice.org.

LibreOffice is offered in a free and open source "Community" version officially intended for personal use. This is the full suite, not a cut-down version. Enterprise-supported versions of LibreOffice can also be obtained from TDF's corporate partners.

The project was announced and a beta released on 28 September 2010. Between January 2011 (the first stable release) and October 2011, LibreOffice was downloaded approximately 7.5 million times. The project claims 120 million unique downloading addresses from May 2011 to May 2015, excluding Linux distributions, with 55 million of those being from May 2014 to May 2015."

**Comment by mayrinck** [comment, marked as answer]
Great, it will be added as soon as possible! Thank you for the suggestion!

**Reply by JuliaAgostino** [reply]
> Great, it will be added as soon as possible! Thank you for the suggestion!

Has any progress been made so far?
